### PR TITLE
fixes rpi repo in kickstart not working + change size for AlmaLinux 8 Gnome

### PR DESCRIPTION
**Pull request by AlexBaranowski**
Siemanko! (eng. Howdy!),

This PR introduces the following changes:

- Fix wrong (IMO) parameters for the raspberrypi repo in kickstarts for Gnome versions. Without this patch, the build fails on Fedora 36 ARM. Now parameters are the same in console and gnome. 
- Change the default size of partitions for Gnome versions. It was set to 3000 MB. At least for AlmaLinux 9 gnome, it was not enough, the build failed with "no space left on device". The 4 GB cards are very rare so this change should not impact users. At the moment 32 GB cards are cheaper than 16GB.  - https://forums.raspberrypi.com/viewtopic.php?t=252380

 Best,
 Alex

### CODE REVIEW

**Comment by psgreco** [comment]
The repo fix looks like a typo initially, so I completely agree.
Wrt the size changes, the problem is not so much the SD card size, but the size of the compressed image to download and the time it takes to `dd` that image, so the theory is always to try to keep the image as small as possible, since it can be expanded on the first boot.
Related to that, I'd really like to eventually remove the swap partition and use a swap file and zram, to make the image even smaller

File: AlmaLinux-8-RaspberryPi-gnome.aarch64.ks [hunk 1]
```diff
@@ -26,9 +26,9 @@ lang en_US.UTF-8
 
 # Disk setup
 clearpart --initlabel --all
-part /boot --asprimary --fstype=vfat --size=500 --label=boot
```
Comment (srbala) [inline AlmaLinux-8-RaspberryPi-gnome.aarch64.ks:29]: Please remove the size change, it cannot be accepted, to keep the size of download image small.

File: AlmaLinux-8-RaspberryPi-gnome.aarch64.ks [hunk 2]
```diff
@@ -11,7 +11,7 @@ rootpw --plaintext almalinux
 # Repositories to use
 repo --name="baseos"    --baseurl=https://repo.almalinux.org/almalinux/8/BaseOS/aarch64/os/
 repo --name="appstream" --baseurl=https://repo.almalinux.org/almalinux/8/AppStream/aarch64/os/
-repo --name="raspberrypi" --baseurl=https://repo.almalinux.org/almalinux/8/raspberrypi/aarch64/os/ --cost=1000 --installr/ --cost=1000 --install
```
Comment (srbala) [inline AlmaLinux-8-RaspberryPi-gnome.aarch64.ks:14]: typo fix, this is ok.

File: AlmaLinux-9-RaspberryPi-gnome.aarch64.ks [hunk 3]
```diff
@@ -26,9 +26,9 @@ lang en_US.UTF-8
 
 # Disk setup
 clearpart --initlabel --all
-part /boot --asprimary --fstype=vfat --size=500 --label=boot
```
Comment (srbala) [inline AlmaLinux-9-RaspberryPi-gnome.aarch64.ks:29]: Remove size change, same as other

**Review by srbala — CHANGES_REQUESTED** [review]

**Comment by AlexBaranowski** [comment]
@srbala I will determine the minimal size for the AL9-gnome to work - give me 15 minutes.

**Comment by srbala** [comment]
Minimal size + 200-500MB is good for the initial startup and sessions. This allows the user to run partition resize/expand.

Would you be able to help us with our PPI enhancements work?

File: AlmaLinux-9-RaspberryPi-gnome.aarch64.ks [hunk 4]
```diff
@@ -28,7 +28,7 @@ lang en_US.UTF-8
 clearpart --initlabel --all
 part /boot --asprimary --fstype=vfat --size=500 --label=boot
 part swap --asprimary --fstype=swap --size=100 --label=swap
-part / --asprimary --fstype=ext4 --size=3200 --label=rootfs
+part / --asprimary --fstype=ext4 --size=4400 --label=rootfs
```
Comment (AlexBaranowski) [inline AlmaLinux-9-RaspberryPi-gnome.aarch64.ks:31]: We have to resize :(.

**Comment by AlexBaranowski** [comment]
@srbala After changes, the PR does the following:

- repo fix
- fixes on AlmaLinux 8 gnome (sorry I thought that it was 9) that requires a bigger partition
Screenshot 2022-12-30 at 14 50 35
 - add .gitignore that ignores cache and builds directories
 - chmod 755 build_al8_gnome
 
 
 @psgreco thanks for Your input. I should not change partition size when it's not necessary.
 
@srbala I would love to help with enhancements.

**Pull request merged**